src/Components/About/About.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import Footer from '../Footer/Footer.jsx'
+import SignUp from '../MainPage/SignUp/SignUp.jsx'
+
+
+
+function About() {
+  return (
+    <div >
+        <h1>About</h1>
+        <span>Why we are so into optmization</span>
+        <span>Body text block Body text block Body text block
+         Body text block Body text block Body text block Body text block Body text
+          block Body text block Body text blockBody text block Body text block
+          Body text block Body text block Body text block Body text block Body
+          ext block </span>
+
+        <Footer page={'ABOUT'} />
```

Comment:
What is that? 
Better prepare some high level handler than this because it is not readable enough. Code should be like a good book ;)